pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -505,6 +506,14 @@ public static void main(String[] args) throws Exception {
 
             reportHistogram.reset();
             oldTime = now;
+
+            if (arguments.testTime > 0) {
+                if (now > testEndTime) {
+                    log.info("------------------- DONE -----------------------");
+                    PerfClientUtils.exit(0);
+                    thread.interrupt();
```

Comment:
`thread.interrupt()` this line should never be called.

Reply:
Good catch, I'll remove the lines.